vcd/resource_vcd_vapp_vm_singlenetwork_test.go

```diff
@@ -185,7 +186,7 @@ func TestAccVcdVAppVmSingleNIC(t *testing.T) {
 				Config: configTextStep10,
 				Check: resource.ComposeAggregateTestCheckFunc(
 					testAccCheckVcdVAppVmExists(netVappName, netVmNameBothNetworks, "vcd_vapp_vm."+netVmNameBothNetworks, &vapp, &vm),
-					resource.TestCheckResourceAttr("vcd_vapp_vm."+netVmNameBothNetworks, "ip", "11.10.0.152"),
+					resource.TestMatchResourceAttr("vcd_vapp_vm."+netVmNameBothNetworks, "ip", regexp.MustCompile(`^11.10.0.152|11.10.2.51*$`)),
```

Comment:
What id the purpose of this regular expression?
It will match `11.10.0.152`, `11x10x0x152`, `11.10.2.5`, `11.10.2.51`, `11.10.2.511`, and more

Reply:
purpose to match 11.10.0.152 or 11.10.2.51

Reply:
this one reverted back, and left to be solved in another PR